mslice/presenters/quick_options_presenter.py

```diff
@@ -39,6 +41,7 @@ def _run_quick_options(view, update_model_function, *args):
 
 
 def _set_axis_options(view, target, model, has_logarithmic, grid):
+    print('setting axis options')
```

Comment:
Is this a left over debugging statement?

Reply:
Yes. Doh! I thought I removed it but I guess I forgot to do `git add` before `git commit --amend`...

Reply:
The print statement was because I couldn't seem to get PyCharm to stop at break points when running the tests through `nose` - not sure why...